application/models/Survey.php

```diff
@@ -1615,7 +1615,6 @@ public function search()
         // Search filter
         $sid_reference = (Yii::app()->db->getDriverName() == 'pgsql' ? ' t.sid::varchar' : 't.sid');
         $aWithRelations[] = 'owner';
-        $aWithRelations[] = 'surveygroup';
```

Comment:
Needed (test by @maziminke )

Reply:
But need another alias `Syntax error or access violation: 1066 Not unique table/alias: 'surveygroup'.`